unuran/src/methods/unur_methods.h

```diff
@@ -0,0 +1,128 @@
+/*****************************************************************************
+ *                                                                           *
+ *          UNURAN -- Universal Non-Uniform Random number generator          *
+ *                                                                           *
+ *****************************************************************************
+ *                                                                           *
+ *   FILE: unur_methods.h                                                    *
+ *                                                                           *
+ *   PURPOSE:                                                                *
+ *         defines bitmasks to identify used method in generator objects     *
+ *                                                                           *
+ *****************************************************************************
+ *                                                                           *
+ *   Copyright (c) 2000-2011 Wolfgang Hoermann and Josef Leydold             *
+ *   Department of Statistics and Mathematics, WU Wien, Austria              *
+ *                                                                           *
+ *   This program is free software; you can redistribute it and/or modify    *
+ *   it under the terms of the GNU General Public License as published by    *
```

Comment:
another reference to GPL: however, the information in this file is largely taken from `unuran/src/methods/unur_methods_source.h`

Reply:
I think the authors have written this. So, it should not be a problem to use this. Should we ask them if it's safe to use and we can remove this reference to GPL?

Reply:
I think it is safe to use it since it is written by the UNU.RAN authors